### ISSUE TITLE
why my result is a white picture

### ISSUE THREAD

**Issue by KarasuLS** [milesial/Pytorch-UNet#500]
when I use my own dateset,the score is very good，but the result is a white picture.
my mask:
![image](https://github.com/milesial/Pytorch-UNet/assets/86554133/67425b8f-e3da-4cd7-81d4-cfdaf39dd444)
training:
![1111](https://github.com/milesial/Pytorch-UNet/assets/86554133/52c0a602-13e1-49c5-9ff1-aa5d21cdf21a)
result:
![image](https://github.com/milesial/Pytorch-UNet/assets/86554133/90a13c30-21a7-41e5-83eb-d66f8c8c7cac)

**Comment by CcCHAN546** [comment]
Hi, I also have this problem. Have you found the reason?

**Comment by wjj1wjj** [comment]
Just lower the learning rate. It's a gradient explosion.
